### ISSUE TITLE
Shadows in Heavy&Colder Rain

### ISSUE THREAD

**Issue by BenKenobie31415** [ComplementaryDevelopment/ComplementaryReimagined#121]
When choosing Heavy&Colder Rainy weather style, it would be nice if the shadows where less noticeable since sharp and clear shadows in overcast weather are a bit odd. I don't know how complex this would be, just a suggestion from an atmosphere-point of view.
![2024-07-24_11 47 26](https://github.com/user-attachments/assets/64d2a5d4-2c8e-4529-bbeb-96d45e36cccb)

**Comment by EminGT** [comment]
Is this a screenshot taken with default settings? It shouldn't be that bad

**Comment by BenKenobie31415** [comment]
No, this is not on default but on [these](https://pastebin.com/wa3QweXJ) settings.

**Comment by EminGT** [comment]
You reduced rain atmosphere intensity while kept the rain lighting intensity the same. This is the reason it looks unbalanced

**Comment by BenKenobie31415** [comment]
Even when I set the Lighting Intesity Multiplier for Rainy Weather back to the default settings, it looks like this
![image](https://github.com/user-attachments/assets/f7ebeedd-48d1-4488-91db-1fe8dd1bf51b)

**Comment by EminGT** [comment]
r5.7 is releasing in a week or so, and I made it so that disabling the "Sun/Moon During Rain" setting now significantly reduces the visibility and sharpness of the rain-time shadows. And setting the rainy weather style to "Heavy & Colder" reduces it a little bit more as well.
